packages/react-composites/src/composites/ChatComposite/file-sharing/FileCardGroup.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { mergeStyles, Stack } from '@fluentui/react';
+import React from 'react';
+
+/**
+ * @beta
+ */
+export interface FileCardGroupProps {
+  children: React.ReactNode;
+}
+
+/**
+ * @beta
+ */
+export const FileCardGroup = (props: FileCardGroupProps): JSX.Element => {
+  const { children } = props;
+  return (
+    <Stack
+      horizontal
+      className={mergeStyles({
```

Comment:
Please define constants for all styles introduced here. Much easier to chase them that way ;)